lib/github.js

```diff
@@ -1,42 +1,80 @@
-const shell = require('shelljs');
-const chalk = require('chalk');
-const git = require('./git');
+const shell = require("shelljs");
+const chalk = require("chalk");
+const git = require("./git");
+const { Octokit } = require("@octokit/core");
 
 // VALIDATIONS
 const validateGithubCredentials = (username, token) => {
   if (!username || !token) {
-    if(!username) {
+    if (!username) {
       console.log(chalk.red("Missing GitHub username. Add it to config:"));
       console.log("mint config -g <username>");
     }
     if (!token) {
-      console.log(chalk.red("Missing GitHub personal access token. Add it to config:"));
+      console.log(
+        chalk.red("Missing GitHub personal access token. Add it to config:")
+      );
       console.log("mint config -t <token>");
-      console.log(chalk.cyanBright("Create a GitHub personal access token using the link below. Give the token full scopes for 'repo' and 'user'. Copy this token and set it with 'mint config -t <token>' "));
+      console.log(
+        chalk.cyanBright(
+          "Create a GitHub personal access token using the link below. Give the token full scopes for 'repo' and 'user'. Copy this token and set it with 'mint config -t <token>' "
+        )
+      );
       console.log("https://github.com/settings/tokens");
-      console.log(chalk.red("*Connection type defaults to 'ssh'. You can change this preference to 'https' by running 'mint config -H'"));
+      console.log(
+        chalk.red(
+          "*Connection type defaults to 'ssh'. You can change this preference to 'https' by running 'mint config -H'"
+        )
+      );
     }
-    return true
+    return true;
   }
-  return false
-}
+  return false;
```

Comment:
this is very confusing. idiomatically should return `true` if its good and `false` if its bad. instead it's reversed.

Reply:
Changed to process.exit(1)